.github/workflows/release.yml

```diff
@@ -363,6 +364,41 @@ jobs:
                   cd test
                   pnpm moonwall test dev_tanssi
 
+    typescript-tests-mozart:
+        runs-on: ubuntu-latest
+        needs: ["set-tags", "build"]
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+              with:
+                  ref: ${{ needs.set-tags.outputs.git_ref }}
+            - name: Pnpm
+              uses: pnpm/action-setup@v3.0.0
+              with:
+                  version: 8
+            - name: "Download binaries"
+              uses: actions/download-artifact@v4
+              with:
+                  name: binaries
+                  path: target/release
+            - name: "Setup Node"
+              uses: actions/setup-node@v4
+              with:
+                  node-version: 20.x
+                  cache: "pnpm"
+            # - name: "Generate typescript api"
+            #   run: |
+            #       chmod uog+x target/release/tanssi-node
+            #       cd typescript-api
+            #       pnpm install
+            #       pnpm create-local-interfaces
+            - name: "Run Dev Relay tests"
+              run: |
+                  chmod uog+x target/release/tanssi-relay
+                  cd test
+                  pnpm install
+                  pnpm moonwall test dev_tanssi_relay
```

Comment:
🤔 Currently failing but works locally now

Reply:
I would not add anything to the CI yet tbh

Reply:
Reverted the change for now